test/unit/setup/node.js

```diff
@@ -14,10 +14,10 @@ if (!global.document || !global.window) {
   var jsdom = require('jsdom').jsdom;
```

Comment:
I do think one of the strengths of the boilerplate is that you don't have to maintain the require list in a setup and a runner.html file.   And multiple times a new test will get added but we'll neglect the runner.html.  That said, I don't think that needs to be a requirement of this PR..